index.js

```diff
@@ -1,6 +1,10 @@
 const Alexa = require('ask-sdk-core');
 
-//const randomDessert = ['tiramisu', 'cheesecake', 'banana pudding', 'boston creme pie']
+const randomDessert = ['Boston Creme Pie', 'Ice Cream', 'Cake', 'Cheesecake']
+const randomDessertGenerator = Math.floor(Math.random() * randomDessert.length)
+const randomDessertAnswer = randomDessert[randomDessertGenerator]
+
+//const randomDessert = {['Boston Creme Pie', 'Ice Cream', 'Cake', 'Cheesecake']
```

Comment:
Always clean up commented out code, I know you weren't totally done here, just a reminder to keep an eye out for it. It's helpful to have when working but we don't want it lingering in GitHub. Think of comments as dev tools that we don't commit.